Sources/Apollo/RequestCreator.swift

```diff
@@ -52,6 +61,26 @@ extension RequestCreator {
     return body
   }
 
+  /// Creates multi-part form data to send with a request
+  ///
+  /// - Parameters:
+  ///   - operation: The operation to create the data for.
+  ///   - files: An array of files to use.
+  ///   - sendOperationIdentifiers: True if operation identifiers should be sent, false if not.
+  ///   - serializationFormat: The format to use to serialize data.
+  /// - Returns: The created form data
+  /// - Throws: Errors creating or loading the form  data
+  public func requestMultipartFormData<Operation: GraphQLOperation>(for operation: Operation,
+                                                                    files: [GraphQLFile],
+                                                                    sendOperationIdentifiers: Bool,
+                                                                    serializationFormat: JSONSerializationFormat.Type) throws -> MultipartFormData {
+    return try requestMultipartFormData(for: operation,
+                                        files: files,
+                                        sendOperationIdentifiers: sendOperationIdentifiers,
+                                        serializationFormat: serializationFormat,
+                                        manualBoundary: nil)
```

Comment:
this also seems like something that could be done by having outside callers choose whether to pass nil or not

Reply:
Will remove this method